### PR TITLE
feat:create filterBar and sample video card

### PR DESCRIPTION
**Pull request by mariyasada**
creating filter bar for and Demo video card for library
sample

### CODE REVIEW

File: src/Components/FilterBar/FilterBar.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./FilterBar.css";
+
+export const FilterBar = () => {
+  return (
+    <div className="all-filter-conatiner flex-center">
+      <ul className="filter-item-container flex-center">
+        <li className="filter-item">All</li>
+        <li className="filter-item">Resin Art</li>
```
Comment (Raghav888) [inline src/Components/FilterBar/FilterBar.jsx:9]: You can craete array and map over it

**Review by Raghav888 — APPROVED** [review]

File: src/Components/Videocard/VideoCard.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.video-card-container {
+  height: 26rem;
+  width: 30rem;
+  flex-direction: column;
+  border: 2px solid var(--light-stone);
+  margin-left: 20rem;
+  margin-top: 12rem;
```
Comment (t007rushi) [inline src/Components/Videocard/VideoCard.css]: can be a one-liner ```margin: 12rem 0 0 12rem ```

File: src/Components/Videocard/VideoCard.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+.avatar-image-container {
+  margin-left: 9px;
+  object-fit: contain;
+  align-self: flex-start;
+}
+.video-title-autor-detail-container {
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 3px;
+  /* background-color: aqua; */
```
Comment (t007rushi) [inline src/Components/Videocard/VideoCard.css]: remove comments

File: src/Components/Videocard/VideoCard.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+.dot-icon {
+  font-size: 2rem;
+  margin-right: -8px;
+}
+.more-info-dots-container {
+  font-size: 2rem;
+  align-self: flex-start;
+  margin-left: 9px;
+  margin-top: 1px;
```
Comment (t007rushi) [inline src/Components/Videocard/VideoCard.css]: same as above

File: src/Components/Videocard/VideoCard.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+}
+.level-of-video {
+  gap: 7px;
+  color: var(--light-stone);
+}
+@media only screen and (min-device-width: 320px) and (max-device-width: 880px) and (-webkit-min-device-pixel-ratio: 2) {
+  /* .image-of-video  {
+    height: 25rem;
+    width: 29rem;
+    /* margin-right: 19rem; */
```
Comment (t007rushi) [inline src/Components/Videocard/VideoCard.css]: same as above

File: src/Pages/VideoListPage/VideoListPage.css [hunk 6]
```diff
@@ -1,3 +1,17 @@
 .Video-page-list-container {
-  /* background-color: blue; */
+  margin-top: 9.2rem;
+  height: 100vh;
+}
+.filter-video-container {
+  flex-direction: column;
```
Comment (t007rushi) [inline src/Pages/VideoListPage/VideoListPage.css]: this can be added to utility class

File: src/Components/FilterBar/FilterBar.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  color: var(--primary-color);
+  border: 2px solid var(--primary-dark-color);
+  border-radius: 2rem;
+  font-weight: 530;
+}
+.filter-link-active {
+  background-color: var(--primary-dark-color);
+  color: var(--white-color);
+}
+@media only screen and (min-device-width: 320px) and (max-device-width: 880px) and (-webkit-min-device-pixel-ratio: 2) {
```
Comment (t007rushi) [inline src/Components/FilterBar/FilterBar.css:30]: Liked that you added responsiveness from start

**Review by t007rushi — COMMENTED** [review]
Looks Good just go through some changes suggested

**Review by t007rushi — APPROVED** [review]

**Pull request merged**